07-http_gordle/final/api/game.go

```diff
@@ -0,0 +1,23 @@
+package api
```

Comment:
I would merge this with the `session` package, as they are highly dependent on each other and keeping them separate makes code more difficult to modify.